Task 3/system/Coding/src/app/GlobalTimers.c

```diff
@@ -20,9 +21,9 @@
 
 /* VARIABLES **************************************************************************************/
 
-SoftTimer timer1;
-SoftTimer timer2;
-SoftTimer timer3;
+static SoftTimer timer1;
+static SoftTimer timer2;
+static SoftTimer timer3;
```

Comment:
- [ ] not according to coding style guide

```suggestion
static SoftTimer gTimer1;
static SoftTimer gTimer2;
static SoftTimer gTimer3;
```